### ISSUE TITLE
Feature: Add "Sort" to Characters screen

### ISSUE THREAD

**Issue by zeg-io** [emfmesquita/beyondhelp#26]
While I generally like having characters sorted alphabetically, I would like to toggle that and most recent.  Because I have an insane amount of characters built as the GM, it is sometimes difficult to find them.

Proposal: Add a button which alternates between "Alphabetical" and "Date Created" maybe with a filter symbol in front of it.  All it would do is bypass your alphabetical sort if on Date Created, or enable if on Alphabetical.

It should also remember the last setting.

**Comment by emfmesquita** [comment]
Hi! This feature would be awesome, the hard part of it is that the character creation date is not exposed from DDB. They probably have this info on their database, I tried to check my characters page and even the character json info (used to render the character sheet) but could not find it.

Also add a feature to manually associate a date/priority to characters would be far from ok and boring as hell if the user has too many characters.

To be honest this is exactly one one the things that would be solved if DDB guys had a closer channel with community developers. I know that create a public API is scary, but if it was available probably this feature would be possible.

I hope one day (maybe soon with DDB discord channel) we will achieve this.

If a dev read this and want to spend 2min sneaking a hidden creation date on my characters page I promise I will revisit this issue.

**Comment by zeg-io** [comment]
Agreed that they need a real API.  The reality is that though their front end is JS they are using C#/server side which makes it that much more annoying.  My proposed solution was to either filter alphabetically, or not filter (which defaults to their order created)

**Comment by emfmesquita** [comment]
You are correct indeed. My mistake! In this case this goes to TODO. 

Starting by separating the turn off all in two on options should be easy. And later adding the ui on my character page.
